edx_api/email_settings.py

```diff
@@ -0,0 +1,60 @@
+"""
+API client for interacting with change email settings
+"""
+import logging
+from urllib import parse
+
+log = logging.getLogger(__name__)
+
+
+class ChangeEmailSettings(object):
+    """
+    API client for interacting with change email settings
+    """
+    api_url = "/api/change_email_settings"
+
+    def __init__(self, requester, base_url):
+        self.requester = requester
+        self.base_url = base_url
+
+    def change_settings(self, payload):
+        """
+        Execute the client request to edX endpoint
+        Args:
+            payload (dict): request payload
+        Returns:
+            JSON response (dict)
+        """
+        response = self.requester.post(
+            parse.urljoin(self.base_url, self.api_url),
+            json=payload
+        )
+
+        try:
+            response.raise_for_status()
+        except Exception:
+            log.error(response.json())
+        return response.json()["success"]
```

Comment:
Are we sure we'll always get this response even if the API breaks? Would there be the `success` key present?

In any case, i think the least we should do here is to use `get()` with a default value like:
```
return response.json().get("success", false)
```